docs/reference/configuration-reference/share-settings.md

```diff
@@ -0,0 +1,34 @@
+---
+navigation_title: "Share settings"
+mapped_pages:
+  - https://www.elastic.co/guide/en/kibana/current/share-settings-kb.html
```

Comment:
```suggestion
```
This doesn't seem to exist so I'd just remove it. And we don't even use that property for new pages I think.